src/components/tooltip/Tooltip.scss

```diff
@@ -64,6 +64,11 @@ $tooltip-offset: 4px; /* amount of space between target and tooltip arrow */
         }
     }
 
+    &.is-wide {
```

Comment:
Is this change related to the preview changes or should it be a separate PR? We will need to very cautious about (or even avoid entirely) using BUIE components in Preview SDK, since it can lead to styling issues.